adapters/play/src/main/scala/caliban/PlayAdapter.scala

```diff
@@ -0,0 +1,197 @@
+package caliban
+
+import akka.stream.{ Materializer, OverflowStrategy, QueueOfferResult }
+import akka.stream.scaladsl.{ Flow, Sink, Source, SourceQueueWithComplete }
+import caliban.ResponseValue.{ ObjectValue, StreamValue }
+import caliban.Value.NullValue
+import play.api.http.Writeable
+import play.api.libs.json.Writes
+import play.api.mvc.{ Action, ActionBuilder, AnyContent, PlayBodyParsers, Request, RequestHeader, Result, WebSocket }
+import play.api.mvc.Results.Ok
+import zio.{ CancelableFuture, Fiber, IO, RIO, Ref, Runtime, Schedule, Task, ZIO }
+import zio.clock.Clock
+import zio.duration.Duration
+import scala.concurrent.{ ExecutionContext, Future }
+
+trait PlayAdapter {
+
+  def json: JsonBackend
```

Comment:
I think you can simplify the API and not expose this, instead just use the play json backend directly.

Reply:
done!